### ISSUE TITLE
Minor issue: jnp -> np -> jnp

### ISSUE THREAD

**Issue by SimonDufLab** [ioneliabuzatu/grn-control#11]
https://github.com/ioneliabuzatu/grn-control/blob/9ec6539eb29e1c7e45c328bc386b4454c80e986c/jax_simulator.py#L85

Very minor issue, but should use jnp.mean instead than np.mean to avoid array conversion from jax to numpy and back to jax again.

Also, (suggestion) I think it would make the code easier to grasp (understand intended behavior) if the mean was taken across axis 0 and 2 (i.e. across cell-type as well) right away here, unless a future feature is to have a mean_expression tensor that varies with cell-type.

----
Same correction can be made here as well:
-[            self.mean_expression = self.mean_expression.at[layer].set(np.mean(self.x[:, layer], axis=0))](https://github.com/ioneliabuzatu/grn-control/blob/9ec6539eb29e1c7e45c328bc386b4454c80e986c/jax_simulator.py#L105)
            -[half_response = np.mean(mean_expression_per_cells_regulators_wise)](https://github.com/ioneliabuzatu/grn-control/blob/9ec6539eb29e1c7e45c328bc386b4454c80e986c/jax_simulator.py#L134)

**Comment by ioneliabuzatu** [comment]
thank you for spotting that, address it here 40799518be2ac9f5ac530b5227466c2a624f6f0e.

about the mean expression that might vary across cell-types its definitely true but i dunno how to model that.
